js/gratipay/edit_team.js

```diff
@@ -0,0 +1,51 @@
+Gratipay.edit_team = {}
+
+Gratipay.edit_team.initForm = function() {
+    $form = $("#edit-team");
+    $buttons = $form.find("button");
+
+    $form.find(":input").each(function() {
+        // Save original values.
+        $(this).data('original', $(this).val());
+    });
+    $form.submit(Gratipay.edit_team.submitForm);
+}
+
+Gratipay.edit_team.submitForm = function(e) {
+    e.preventDefault();
+
+    var modified = false;
+
+    var data = new FormData();
+    $form.find(":input").each(function() {
+        if($(this).data("original") !== $(this).val()) {
+            modified = true;
+
+            var name = $(this).attr("name");
+            if (name === "image") {
+                data.append(name, $(this)[0].files[0]);
+            } else {
+                data.append(name, $(this).val());
```

Comment:
Also, this is supposed to be a `PATCH` not a `PUT` request. The former requires only the attributes being updated to be passed in the request body. I wanted to [use](https://github.com/gratipay/gratipay.com/pull/3923/files#diff-8cd27f320d80ef3f1fed6dc52f5feafdR11) `patch` here, but for some reason it breaks everything. :( The idea here is that if later on we decide to expose this api to public officially, you shouldn't have to do a `GET` (to get all the attributes)  before hitting this endpoint.